src/domain/documentation/DocumentationPage.tsx

```
@@ -8,14 +9,68 @@ import Gutters from '@/core/ui/grid/Gutters';
 import { useConfig } from '../platform/config/useConfig';
 import { TopLevelRoutePath } from '@/main/routing/TopLevelRoutePath';
 import { useTranslation } from 'react-i18next';
+import scrollToTop from '@/core/ui/utils/scrollToTop';
+import useNavigate from '@/core/routing/useNavigate';
+
+const DEFAULT_HEIGHT = '100vh';
+const enum SupportedMessageTypes {
+  PageHeight = 'PAGE_HEIGHT',
+  PageChange = 'PAGE_CHANGE',
+}
```

Biome (1.9.4)

[error] The enum declaration should not be const

Const enums are not supported by bundlers and are incompatible with the 'isolatedModules' mode. Their use can lead to import inexistent values.
See TypeScript Docs for more details.
Safe fix: Turn the const enum into a regular enum.


(lint/suspicious/noConstEnum)